app/controllers/customer_items_controller.rb

```diff
@@ -0,0 +1,2 @@
+class CustomerItemsController < ApplicationController
```

Comment:
You were on the right track creating a joins controller! This is where we would want to add a `create` action that our form can route to in order to create the association between a customer and an item